JoonasHeinonen/escape-from-zordoom-prison
# Feature/negotiator

To test out!
1. pull down the Branch feature/negotiator
2. open up godot and load up the main menu level
3. pick Anglea or Rivet
4. check and see in the weapon wheel that the Negotiator is in the wheel
5. shoot at some boxes and the lamp post to see that everything works!
